site/index.html

```diff
@@ -5,37 +5,48 @@
 ---
 <div class="header">
    <div class=" container ">
-      <div class=" row ">
-         <div class=" span12 col-sm-12 hidden-xs"><br></div>
+      <div class="row" >
+         <div class=" span12 col-sm-12 hidden-xs"><br><br></div>
+      </div>
+      <div class=" row " style="display: flex; align-items: center">
          <div class=" span6 col-md-6">
             <div class="branding">
-               <br class="hidden-xs"><img src="/img/haskell-logo.svg" class="img-responsive">
-               <h4 class="summary">An advanced, purely functional programming language</h4>
+              <br class="hidden-xs">
```

Comment:
Here I just formatted the code to be consistent.

Reply:
That's fine. I think that it would be easier to have that in a separate PR, or not at all. It doesn't seem like a big deal but just makes this PR bigger and harder to understand.